src/main/java/tr/edu/ogu/ceng/model/Student.java

```diff
@@ -148,4 +76,7 @@ public class Student {
 	@JoinColumn(name = "faculty_id", nullable = false)
 	private Faculty faculty;
 
+	@Column(name = "address", nullable = false)
```

Comment:
Adresi bu şekilde tutmak yerine il ilçe olarak tutarsak veri tabanında normalizasyona uygun hale gelir. Hatta belki de sadece il bilgisi de yeterli olur.

Reply:
Genelde il ilçe bilgisi yeterli oluyor diye biliyorum. Okuldaki evrak doldurulurken de böyle yazılması yeterli diye söylemişti Hüseyin hoca ama fakültelere göre değişiklik gösterebilir.

Reply:
@ykartal Hocam bu konudaki en doğru yaklaşım nasıl olmalıdır?

Reply:
il ilçe yeterliyse bile adres olarak kalması resmiyet anlamında daha uygun. Normalizasyon için ileride gerekli yapılar ihtiyaca göre eklene de bilir.